package.json

```diff
@@ -32,5 +34,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-strongloop": "^2.1.0"
```

Comment:
What is the usage of this dependency?

Reply:
I followed [this](https://medium.com/the-node-js-collection/why-and-how-to-use-eslint-in-your-project-742d0bc61ed7) link given in task description, that dependancy is to enforce `StrongLoop` code style